applications/data_analysis/climatology.ipynb

```diff
@@ -0,0 +1,3551 @@
+{
```

Comment:
<p>This is more of a "final polish" item, but we might want to find a way to hide some of this output by default. I think maybe we address that later though and not in this PR.</p><p> </p><p>The serialization warnings would be good to hide or address as well.  Same thoughts there though (i.e. a problem for later)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>